JSDemos/Demos/Scheduler/CellTemplates/Angular/app/app.component.ts

```diff
@@ -4,11 +4,13 @@ import {
 import { BrowserModule, BrowserTransferStateModule } from '@angular/platform-browser';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 import { DxSchedulerModule, DxTemplateModule } from 'devextreme-angular';
-import {
+import dxScheduler, {
```

Comment:
In prev PRs you use dxComponent['instance']. Why do you use component from devextreme?